.github/workflows/Main.yml

```diff
@@ -1,28 +1,47 @@
 name: MainJob
 on:
   push:
-    branches: [ '**' ]
+    branches: ['**']
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        dotnet-version: ['6.0.x' ]
+        dotnet-version: ['6.0.x']
     steps:
-      - uses: actions/checkout@v3
-      - name: Setup dotnet
-        uses: actions/setup-dotnet@v3
-        with:
-          dotnet-version: '6.0.x'
-      - name: Build
-        run: dotnet build TodoFramework.sln
-      - name: Check format issues
-        run: dotnet format --verify-no-changes -v d TodoFramework.sln
+    - uses: actions/checkout@v3
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '6.0.x'
+    - name: Install tools
+      run: dotnet tool install --global SpecFlow.Plus.LivingDoc.CLI
+    - name: Build
+      run: dotnet build TodoFramework.sln
+    - name: Check format issues
+      run: dotnet format --verify-no-changes -v d TodoFramework.sln
+
+    - name: Setup Secrets
+      run: |
+        echo "TODO-LY-EMAIL=${{secrets.TODO_LY_EMAIL}}" >> $GITHUB_ENV
+        echo "TODO-LY-PASS=${{secrets.TODO_LY_PASS}}" >> $GITHUB_ENV
+        echo "DriverMode=Headless" >> $GITHUB_ENV
+    - name: Run UI tests
+      run: dotnet test 3.Tests/UI --no-build --no-restore --filter "Category=Smoke | Category=Regression" --logger "trx;LogFileName=${{ github.run_number }}.trx"
+
+    - name: Create Livingdoc reports
+      if: always()
+      run: livingdoc feature-folder 3.Tests/UI --output 3.Tests/UI/ReportOutputs/LivingDoc.html
+    - name: Upload LivingDoc report file
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: Run#${{ github.run_number }}_LivingDoc.html
+        path: 3.Tests/UI/ReportOutputs/LivingDoc.html
+    - name: Upload .trx report file
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: Run#${{ github.run_number }}_report.trx
+        path: 3.Tests/UI/TestResults/${{ github.run_number }}.trx
```

Comment:
Why do we need to upload the .trx file as an artifact?

Reply:
deleted trx file